poc/vdaf_prio3.sage

```diff
@@ -371,28 +371,42 @@ class Prio3Aes128Histogram(Prio3):
 # TESTS
 #
 
+class Prio3Aes128Average(Prio3Aes128Sum):
+    '''
+    A Prio3 instantiation to test use of num_measurements in the Valid
+    class's decode() method.
+    '''
+
+    @classmethod
+    def with_bits(cls, bits: Unsigned):
+        new_cls = deepcopy(cls).with_prg(prg.PrgAes128)
+        new_cls.Flp = flp_generic.FlpGeneric \
+            .with_valid(flp_generic.Average.with_bits(bits))
+        return new_cls
```

Comment:
Does this actually need to be re-implemented?

Reply:
I re-implemented it to swap out `flp_generic.Sum` for `flp_generic.Average`. I could possibly define some new members on `Prio3Aes128Sum`, and and override a member for the validity circuit instead, but I figure the separate factory classmethod is less invasive.